Add Raspberry Pi Zero Bracket to printer_mods

D4rkCelerity opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [ X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ X] Submitted STL files are oriented and scaled properly for printing.
  * [ n/a] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?
Add Raspberry Pi Zero Bracket to printer_mods
#### How was it tested? 
It was assembled.
#### Any background context you want to provide?
This is a modification of the Pi Bracket from the Trident.  It was resized to work with the RPi Zero form factor
#### Screenshots (if appropriate)
n/a
#### Further notes
n/a

cbc02009 commented on printer_mods/README.md:
```diff
@@ -92,6 +92,7 @@ like so:
 |consibonsi | [Y-Mod SW 9mm](./consibonsi/Y-Mod_SW_9mm)| 9mm Y belt mod for SW | :x: |:x: |:x: |:heavy_check_mark:|
 |csch0 | [SW Fan Mount No Tape](./csch0/SW_FanMount_No_Tape)| 60x20 Fan Mount for SW without tape| :x: |:x: |:x: |:heavy_check_mark:|
 | d3nny | [Filtered Power Plug with RND_165-00057](./d3nny/mains_plug_panel_for_RND_165-00057) | Filtered Power Plug for RND 165-00057 and CAT6-Plug | :x: | :x: | :heavy_check_mark: |:x:|
+|D4rkCelerity| [Raspberry Pi Zero Bracket](./D4rkCelerity/Raspberry_Pi_Zero_Bracket/) | Modified version of Trident RPi bracket to fit the smaller RPi Zero | :x: | :grey_question: | :grey_question: | :x: | :heavy_check_mark: |
```
if you want the check the v0 it should be fine since it can be attached with VHB. You can also check the v2 and v1.8 columns, since they both use DIN rail mounts.

D4rkCelerity commented:
Hello,

I made the changes you noted above.  I also added a note in the mod
file's readme.md about using VHB to install on a V0 and added a line on
where to find the matching DIN clip STL on GitHub.

This is my first time using GitHub so please let me know if you see
anything else to clean up or if I need to do anything else to keep this
pull request moving forward.

Thank you,

On Fri, Dec 31, 2021 at 1:51 PM cbc02009 ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In printer_mods/README.md
> <https://github.com/VoronDesign/VoronUsers/pull/562#discussion_r777058896>
> :
>
> > @@ -92,6 +92,7 @@ like so:
>  |consibonsi | [Y-Mod SW 9mm](./consibonsi/Y-Mod_SW_9mm)| 9mm Y belt mod for SW | :x: |:x: |:x: |:heavy_check_mark:|
>  |csch0 | [SW Fan Mount No Tape](./csch0/SW_FanMount_No_Tape)| 60x20 Fan Mount for SW without tape| :x: |:x: |:x: |:heavy_check_mark:|
>  | d3nny | [Filtered Power Plug with RND_165-00057](./d3nny/mains_plug_panel_for_RND_165-00057) | Filtered Power Plug for RND 165-00057 and CAT6-Plug | :x: | :x: | :heavy_check_mark: |:x:|
> +|D4rkCelerity| [Raspberry Pi Zero Bracket](./D4rkCelerity/Raspberry_Pi_Zero_Bracket/) | Modified version of Trident RPi bracket to fit the smaller RPi Zero | :x: | :grey_question: | :grey_question: | :x: | :heavy_check_mark: |
>
> if you want the check the v0 it should be fine since it can be attached
> with VHB. You can also check the v2 and v1.8 columns, since they both use
> DIN rail mounts.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/VoronDesign/VoronUsers/pull/562#pullrequestreview-842303285>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXC7TGIVXPMBIV5VIC6EYRTUTYQW5ANCNFSM5LALGOAA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

cbc02009 commented on printer_mods/D4rkCelerity/Raspberry_Pi_Zero_Bracket/README.md:
```diff
@@ -2,5 +2,12 @@ Raspberry Pi Zero Bracket
-Use the same type and quantity of fasteners from the BOM to mount the bracket to din clip and then the RPi Zero to the bracket.
+DIN Rail Install:
+
+ The STL file for the pcb_din_clip_v2 is located on GitHub at VoronDesign/Voron-Trident/STLs/ElectronicsBay/
```
Can you make this a hyperlink so that people can click on it and go straight to the necessary part?

cbc02009 reviewed (changes requested):
looks good! one last change and we should be all set.

D4rkCelerity commented:
Hyperlink is added.

On Mon, Jan 3, 2022 at 1:22 PM cbc02009 ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> looks good! one last change and we should be all set.
> ------------------------------
>
> In printer_mods/D4rkCelerity/Raspberry_Pi_Zero_Bracket/README.md
> <https://github.com/VoronDesign/VoronUsers/pull/562#discussion_r777709856>
> :
>
> > @@ -2,5 +2,12 @@ Raspberry Pi Zero Bracket
>
>  This is a modified version of the raspberrypi_bracket for the VORON Trident. Use this bracket along with a pcb_din_clip_v2 to mount a Raspberry Pi Zero to a DIN rail.
>
> -Use the same type and quantity of fasteners from the BOM to mount the bracket to din clip and then the RPi Zero to the bracket.
> +DIN Rail Install:
> +
> + The STL file for the pcb_din_clip_v2 is located on GitHub at VoronDesign/Voron-Trident/STLs/ElectronicsBay/
>
> Can you make this a hyperlink so that people can click on it and go
> straight to the necessary part?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/VoronDesign/VoronUsers/pull/562#pullrequestreview-843049171>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXC7TGPY2VHCLFPWWN3MHBLUUIHP3ANCNFSM5LALGOAA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

D4rkCelerity commented:
Hey, Checking to see if any other changes need to be made.

Thanks,

On Tue, Jan 4, 2022 at 6:30 AM Aaron Rieck ***@***.***> wrote:

> Hyperlink is added.
>
> On Mon, Jan 3, 2022 at 1:22 PM cbc02009 ***@***.***> wrote:
>
>> ***@***.**** requested changes on this pull request.
>>
>> looks good! one last change and we should be all set.
>> ------------------------------
>>
>> In printer_mods/D4rkCelerity/Raspberry_Pi_Zero_Bracket/README.md
>> <https://github.com/VoronDesign/VoronUsers/pull/562#discussion_r777709856>
>> :
>>
>> > @@ -2,5 +2,12 @@ Raspberry Pi Zero Bracket
>>
>>  This is a modified version of the raspberrypi_bracket for the VORON Trident. Use this bracket along with a pcb_din_clip_v2 to mount a Raspberry Pi Zero to a DIN rail.
>>
>> -Use the same type and quantity of fasteners from the BOM to mount the bracket to din clip and then the RPi Zero to the bracket.
>> +DIN Rail Install:
>> +
>> + The STL file for the pcb_din_clip_v2 is located on GitHub at VoronDesign/Voron-Trident/STLs/ElectronicsBay/
>>
>> Can you make this a hyperlink so that people can click on it and go
>> straight to the necessary part?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/VoronDesign/VoronUsers/pull/562#pullrequestreview-843049171>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AXC7TGPY2VHCLFPWWN3MHBLUUIHP3ANCNFSM5LALGOAA>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

D4rkCelerity commented:
Hello - Checking to see if any follow up is needed.

Thanks,

On Mon, Jan 10, 2022 at 8:25 PM Aaron Rieck ***@***.***> wrote:

> Hey, Checking to see if any other changes need to be made.
>
> Thanks,
>
> On Tue, Jan 4, 2022 at 6:30 AM Aaron Rieck ***@***.***> wrote:
>
>> Hyperlink is added.
>>
>> On Mon, Jan 3, 2022 at 1:22 PM cbc02009 ***@***.***> wrote:
>>
>>> ***@***.**** requested changes on this pull request.
>>>
>>> looks good! one last change and we should be all set.
>>> ------------------------------
>>>
>>> In printer_mods/D4rkCelerity/Raspberry_Pi_Zero_Bracket/README.md
>>> <https://github.com/VoronDesign/VoronUsers/pull/562#discussion_r777709856>
>>> :
>>>
>>> > @@ -2,5 +2,12 @@ Raspberry Pi Zero Bracket
>>>
>>>  This is a modified version of the raspberrypi_bracket for the VORON Trident. Use this bracket along with a pcb_din_clip_v2 to mount a Raspberry Pi Zero to a DIN rail.
>>>
>>> -Use the same type and quantity of fasteners from the BOM to mount the bracket to din clip and then the RPi Zero to the bracket.
>>> +DIN Rail Install:
>>> +
>>> + The STL file for the pcb_din_clip_v2 is located on GitHub at VoronDesign/Voron-Trident/STLs/ElectronicsBay/
>>>
>>> Can you make this a hyperlink so that people can click on it and go
>>> straight to the necessary part?
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/VoronDesign/VoronUsers/pull/562#pullrequestreview-843049171>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AXC7TGPY2VHCLFPWWN3MHBLUUIHP3ANCNFSM5LALGOAA>
>>> .
>>> Triage notifications on the go with GitHub Mobile for iOS
>>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>>> or Android
>>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>>
>>> You are receiving this because you authored the thread.Message ID:
>>> ***@***.***>
>>>
>>
